sdcm/nemesis.py

```diff
@@ -4205,7 +4205,12 @@ def disrupt_add_remove_mv(self):
         Verify the MV can be used in a query.
         Finally, drop the MV.
         """
-        node1, node2 = self.cluster.nodes[:2]
+        try:
+            node1, node2 = [node for node in self.cluster.nodes if not node.running_nemesis][:2]
```

Comment:
i'm not sure i understand the problem, but looks like you want to choose a node to be a target node for this nemesis, that doesn't have another nemesis running in it... and if that is the case, why not to:
```
self.target_node = random.choice([node for node in self.cluster.nodes if not node.running_nemesis])
```